moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -134,6 +134,18 @@ export const ProjectsListViewQueryConf = {
       width: "20%",
       filter: filterProjectTeamMembers,
     },
+    project_lead: {
+      label: "Project lead",
+      searchable: true,
```

Comment:
since we are adding this to the search too, should we expand the prompt to include Project lead?
![Screen Shot 2022-11-09 at 10 32 41 AM](https://user-images.githubusercontent.com/12474808/200886894-aa7dbe8f-c613-41d6-a666-61c497ca5ebc.png)

Reply:
good point @chiaberry, i might have gone overboard with this pr because looking back i don't think adding this field to global search was part of the issue. i should either remove that or add it to this prompt as you suggested. @johnclary what are your thoughts?

Reply:
TIL we can currently search by sponsor and partners. I like this so i updated the placeholder text. Thank you both!